images/distros/centos-7

```diff
@@ -3,5 +3,5 @@ IMAGE_NAME=centos-7
 
 function prepare {
     virt-sysprep -a ${UPSTREAM_IMAGE_DIR}/${IMAGE_NAME}.qcow2 \
-        --network --update --install NetworkManager --run-command 'systemctl enable NetworkManager' --selinux-relabel
+        --network --run-command 'echo "nameserver 169.254.2.3" > /etc/resolv.conf && yum update -y && yum install -y NetworkManager' --run-command 'systemctl enable NetworkManager' --selinux-relabel
```

Comment:
Why is nameserver local? I think better use 8.8.8.8 than 169.254.2.3

Reply:
It's very special to SLIRP https://wiki.qemu.org/Documentation/Networking#User_Networking_.28SLIRP.29

SLIRP allow only route TCP connections, this is the reason why qemu also provides a local DHCP server.